src/components/data-table/config/index.js

```diff
@@ -0,0 +1,74 @@
+export const args = {
+  hover: false,
+  matrix: false,
+  lined: false,
+  zebra: false,
+  collapsedM: false,
+  collapsedS: false,
+  collapsedXS: false,
+};
+
+export const argTypes = {
```

Comment:
We geven nu ook aan of het een attribuut/prop/event, .. is. zoals bv hier: https://github.com/milieuinfo/uig-webcomponents/blob/master/src/components/progress-bar/progress-bar.stories.js

Reply:
Fixed.